css/components/card.css

```diff
@@ -0,0 +1,178 @@
+.display-flex-column {
```

Comment:
give naming convention according to your component that you are making instead of property name, this convention is hard to understand. You can do this naming in utils